src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx

```diff
@@ -126,6 +126,12 @@ const GroupDetails = createReactClass({
           Sentry.withScope(scope => {
             Sentry.captureException(new Error('Project not found'));
           });
+        } else {
+          const locationWithProject = {...browserHistory.getCurrentLocation()};
+          if (!locationWithProject.query.project) {
+            locationWithProject.query.project = project.id;
+            browserHistory.replace(locationWithProject);
```

Comment:
Is this going to cause an additional re-render?

Reply:
The page is barely rendered at the point in time of replace. It changes before it loads page content, so this may not be an issue.

There would be a need to re-render certain components that rely on it for link properly though, right?

Reply:
Ugh, the following is kind of hacky, but if we're going to update the URL like this, maybe the additional URL parameter is the way to go.

So if you have multi project support and "all projects" is selected, we navigate to the issue details with a URL parameters like `_allp=1` and in group details remove that query param when we do the above.